tests/testdata/expected_addons/expected_output_global_config_configuration/Splunk_TA_UCCExample/bin/splunk_ta_uccexample_rh_settings.py

```diff
@@ -14,6 +14,10 @@
 util.remove_http_proxy_env_vars()
 
 
+special_fields = [
```

Comment:
Can we skip the declaration of special_fields if it is an empty list?

Reply:
like above: "we can, but how do we benefit from it? I think it's not worth adding extra logic and template just to avoid declaring an empty list. What do you think about it?"